plugins/datastore.js

```diff
@@ -24,6 +24,10 @@ const SCHEMA_SCAN = Joi.object().keys({
         count: Joi.number().integer().positive().required(),
         page: Joi.number().integer().positive().required()
     }),
+    search: Joi.object().keys({
+        field: Joi.string().max(100).required(),
+        term: Joi.string().max(200).required()
```

Comment:
`field` and `term` sound similar to me. What is `term`? Is it the value to search for? Would `value` be less confusing?

Reply:
I guess search `term` or search `keyword` captures what I mean the most (https://www.opentracker.net/article/choosing-search-terms), but when those words are compared with `field`, it is confusing.

Would `key` and `value` be clearer? `field` and ` value`?

Reply:
I think `key/value` or `field/keyword` would make sense.
I'm leaning toward `key/value` because it's easy to understand...

Reply:
Changed to `field/keyword` 👍